src/runner/settings_window.cpp

```diff
@@ -41,7 +43,15 @@ json::JsonObject get_all_settings() {
 void dispatch_json_action_to_module(const json::JsonObject& powertoys_configs) {
   for (const auto& powertoy_element : powertoys_configs) {
     const std::wstring name{powertoy_element.Key().c_str()};
-    if (modules().find(name) != modules().end()) {
+    if (name == L"general") {
```

Comment:
it's not clear why we're restarting stuff here when the `name == L"general"` is true

Reply:
I'll add a comment in the code.